src/libraries/pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -1,98 +0,0 @@
-    <PrereleaseLibraryPackage Include="System.Diagnostics.PerformanceCounter" />
-    <PrereleaseLibraryPackage Include="System.DirectoryServices" />
-    <PrereleaseLibraryPackage Include="System.DirectoryServices.AccountManagement" />
-    <PrereleaseLibraryPackage Include="System.DirectoryServices.Protocols" />
-    <PrereleaseLibraryPackage Include="System.IO.Packaging" />
-    <PrereleaseLibraryPackage Include="System.IO.Ports" />
-    <PrereleaseLibraryPackage Include="System.Management" />
-    <PrereleaseLibraryPackage Include="System.Runtime.Caching" />
-    <PrereleaseLibraryPackage Include="System.Security.Cryptography.Pkcs" />
-    <PrereleaseLibraryPackage Include="System.Security.Cryptography.ProtectedData" />
-    <PrereleaseLibraryPackage Include="System.Security.Cryptography.Xml" />
-    <PrereleaseLibraryPackage Include="System.Security.Permissions" />
-    <PrereleaseLibraryPackage Include="System.ServiceModel.Syndication" />
-    <PrereleaseLibraryPackage Include="System.ServiceProcess.ServiceController" />
-    <PrereleaseLibraryPackage Include="System.Speech" />
-    <PrereleaseLibraryPackage Include="System.Text.Encoding.CodePages" />
-    <PrereleaseLibraryPackage Include="System.Threading.AccessControl" />
-
-    <NS21PrereleaseLibraryPackage Include="System.ComponentModel.Composition.Registration" />
-    <NS21PrereleaseLibraryPackage Include="System.Reflection.Context" />
-
-    <!-- Packages which are inbox in NET6 and don't need to be referenced. -->
-    <BeforeNET6LibraryPackage Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryVersion)" />
-    <BeforeNET6LibraryPackage Include="System.Data.DataSetExtensions" Version="$(SystemDataDataSetExtensionsVersion)" />
-    <BeforeNET6LibraryPackage Include="System.IO.FileSystem.AccessControl" Version="$(SystemIOFileSystemAccessControlVersion)" />
-    <BeforeNET6LibraryPackage Include="System.IO.Pipes.AccessControl" Version="$(SystemIOPipesAccessControlVersion)" />
-    <BeforeNET6LibraryPackage Include="System.Reflection.Emit" Version="$(SystemReflectionEmitVersion)" />
-    <BeforeNET6LibraryPackage Include="System.Reflection.Emit.ILGeneration" Version="$(SystemReflectionEmitILGenerationVersion)" />
-    <BeforeNET6LibraryPackage Include="System.Reflection.Emit.Lightweight" Version="$(SystemReflectionEmitLightweightVersion)" />
-    <BeforeNET6LibraryPackage Include="System.Security.AccessControl" Version="$(SystemSecurityAccessControlVersion)" />
-    <BeforeNET6LibraryPackage Include="System.Security.Cryptography.Cng" Version="$(SystemSecurityCryptographyCngVersion)" />
-    <BeforeNET6LibraryPackage Include="System.Security.Principal.Windows" Version="$(SystemSecurityPrincipalWindowsVersion)" />
-
-    <!-- External packages -->
-    <ExternalLibraryPackage Include="System.Data.SqlClient" Version="$(SystemDataSqlClientVersion)" />  
-    <ExternalLibraryPackage Include="System.ServiceModel.Primitives;
```

Comment:
these are missing from the package in the diff for net6.0

Reply:
Those were missing from the P6 package. In the diff the left side is the P6 package, the right side is the new package.

Reply:
okay so thats an intentional diff ?